src/site/apt/examples/control-dependencies-under-dependency-analysis.apt.vm

```diff
@@ -37,6 +37,11 @@ Exclude dependencies from dependency analysis
   The plugin can then be configured to ignore dependencies that are
   "declared but unused", "undeclared but used", and "non-test scoped"
   in selected list or in all simultaneously.
+  
+  Another case is when mature projects have accumulated superfluous dependencies or
+  relied upon transitivity for their dependencies. Where fixing all the dependency 
+  problems up-front is not viable the plugin can be configured to only raise
+  warnings for dependencies meeting "include" criteria.
```

Comment:
This needs to be expanded. Based solely on this text — that is, without reading the code or the JIRA — I do not know which dependencies are and are not checked.